<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite ignored="1" hostname="cgvak-lt177" failures="0" tests="1" name="com.opemmrs.testcase.TC_Register" time="43.192" errors="1" timestamp="06 May 2023 09:45:06 GMT" skipped="0">
  <testcase classname="com.opemmrs.testcase.TC_Register" name="PatientName" time="43.192">
    <error message="Proxy element for: DefaultElementLocator &#039;By.xpath: //input[@id=&#039;fr3813-field&#039;]&#039; Element is still not ready to pass the text Test" type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException: Proxy element for: DefaultElementLocator 'By.xpath: //input[@id='fr3813-field']' Element is still not ready to pass the text Test
	at com.opemmrs.testcase.BaseClass.passTextToWebelement(BaseClass.java:135)
	at com.opemmrs.testcase.TC_Register.PatientName(TC_Register.java:31)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- PatientName -->
  <testcase classname="com.opemmrs.testcase.TC_Register" name="RegisterPatient">
    <skipped/>
  </testcase> <!-- RegisterPatient -->
</testsuite> <!-- com.opemmrs.testcase.TC_Register -->
